Adapt VC10/MSVC2010 stuff for VC11/MSVC2012 (apparently builds with both then)

diff -ru python3.old_/PCbuild/_msi.vcxproj python3/PCbuild/_msi.vcxproj
--- python3.old_/PCbuild/_msi.vcxproj	2012-09-29 10:00:48.000000000 +0200
+++ python3/PCbuild/_msi.vcxproj	2012-11-13 15:52:40.656101165 +0100
@@ -151,7 +151,7 @@
   </PropertyGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <Link>
-      <AdditionalDependencies>fci.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>cabinet.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <BaseAddress>0x1D160000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
@@ -160,13 +160,13 @@
       <TargetEnvironment>X64</TargetEnvironment>
     </Midl>
     <Link>
-      <AdditionalDependencies>fci.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>cabinet.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <BaseAddress>0x1D160000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">
     <Link>
-      <AdditionalDependencies>fci.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>cabinet.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <BaseAddress>0x1D160000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
@@ -175,13 +175,13 @@
       <TargetEnvironment>X64</TargetEnvironment>
     </Midl>
     <Link>
-      <AdditionalDependencies>fci.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>cabinet.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <BaseAddress>0x1D160000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='PGInstrument|Win32'">
     <Link>
-      <AdditionalDependencies>fci.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>cabinet.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <BaseAddress>0x1D160000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
@@ -190,14 +190,14 @@
       <TargetEnvironment>X64</TargetEnvironment>
     </Midl>
     <Link>
-      <AdditionalDependencies>fci.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>cabinet.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <BaseAddress>0x1D160000</BaseAddress>
       <TargetMachine>MachineX64</TargetMachine>
     </Link>
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='PGUpdate|Win32'">
     <Link>
-      <AdditionalDependencies>fci.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>cabinet.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <BaseAddress>0x1D160000</BaseAddress>
     </Link>
   </ItemDefinitionGroup>
@@ -206,7 +206,7 @@
       <TargetEnvironment>X64</TargetEnvironment>
     </Midl>
     <Link>
-      <AdditionalDependencies>fci.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>cabinet.lib;msi.lib;rpcrt4.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <BaseAddress>0x1D160000</BaseAddress>
       <TargetMachine>MachineX64</TargetMachine>
     </Link>
